alcs-api/src/application/application-decision-meeting/application-decision-meeting.service.ts

```diff
@@ -55,4 +56,16 @@ export class ApplicationDecisionMeetingService {
     const meeting = await this.get(uuid);
     return this.appDecisionMeetingRepository.softRemove([meeting]);
   }
+
+  async getUpcomingMeetings(): Promise<
+    { uuid: string; next_meeting: string }[]
+  > {
+    return await this.appDecisionMeetingRepository
+      .createQueryBuilder('meeting')
+      .select('application.uuid, MAX(meeting.date) as next_meeting')
+      .leftJoin('meeting.application', 'application')
```

Comment:
Why left join? If you only care about applications with meetings, then inner is preferable

Reply:
This is joining meeting onto Application which is not nullable and has a FK, leftJoin and innerJoin will return the same thing.

Reply:
Happy to change if its still better.